api/user_test.go

```diff
@@ -2339,3 +2339,81 @@ func TestAutocompleteUsers(t *testing.T) {
 		t.Fatal("should have errored - bad team id")
 	}
 }
+
+func TestGetByUsername(t *testing.T) {
+	th := Setup().InitBasic()
+	Client := th.CreateClient()
```

Comment:
The client can be accessed by doing `th.BasicClient` so no need to do `th.CreateClient()` here